### PR TITLE
Feat: added PDP

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Clicking on a product take user to the Product page , from here user can add and wishlist products

### CODE REVIEW

File: src/context/ProductListingContext.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ const ProductListingProvider = ({ children }) => {
 			}
 		})();
 	}, []);
+	console.log("products in context", productListingState.data);
```
Comment (AbhishekkGautam) [inline src/context/ProductListingContext.js]: We should remove the console.log before commit.

File: src/pages/ProductListing/ProductListing.jsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ export const ProductListing = () => {
 	const categoryProducts = getFliteredProducts(discountedProducts, categories);
 	const ratedProducts = getRatedProducts(categoryProducts, rating);
 	const finalFilteredProducts = getSortedProducts(ratedProducts, sortBy);
+	console.log("products", productListingState.data);
```
Comment (gopalbbharadva) [inline src/pages/ProductListing/ProductListing.jsx:24]: You can remove console if not needed.

**Review by gopalbbharadva — APPROVED** [review]
Bug: on click of add to cart of single product, it navigate to login but not on add to wishlist button, check add to wishlist button.

Suggestion: you can put back button or close button to close single product page and back to product listing.

File: src/pages/ProductListing/ProductListing.jsx (hunk 2 above)
Comment (AbhishekkGautam) [inline src/pages/ProductListing/ProductListing.jsx:24]: We should remove console.log from here also.

**Review by AbhishekkGautam — APPROVED** [review]
Looks good, clean & structured code. 👍

**Pull request merged**